src/AureliaPlugin.ts

```diff
@@ -160,7 +160,9 @@ export class AureliaPlugin {
     if (!dllPlugin && !opts.noWebpackLoader) {
       // Setup aurelia-loader-webpack as the module loader
       // Note that code inside aurelia-loader-webpack performs PLATFORM.Loader = WebpackLoader;
-      this.addEntry(compiler.options, "aurelia-loader-webpack");      
+      // Since this runs very early, before any other Aurelia code, we need "aurelia-polyfills"
+      // for older platforms (e.g. `Map` is undefined in IE 10-).
+      this.addEntry(compiler.options, ["aurelia-polyfills", "aurelia-loader-webpack"]);      
```

Comment:
Does this change mean, that `aurelia-polyfills` must be used instead of alternatives?